### PR TITLE
Upgrade preset to Neutrino v4 compatibility

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,281 @@
...
+- Zero upfront configuration necessary to start developing and building a React web app
+- Modern Babel compilation adding JSX and object rest spread syntax.
+- Support for React Hot Loader
+- Write JSX in .js or .jsx files
+- Extends from [neutrino-preset-web](https://neutrino.js.org/customization/presets/neutrino-preset-web)
+  - Modern Babel compilation supporting ES modules, last 2 major browser versions, and async functions
+  - Webpack loaders for importing HTML, CSS, images, icons, and fonts
+  - Webpack Dev Server during development
+  - Automatic creation of HTML pages, no templating necessary
+  - Hot module replacement support
```
Comment (helfi92) [inline README.md]: 1. Should we make mention of Tree-shaking? It's a pretty huge feature :)
2. Capitalize **H**ot **M**odule **R**eplacement

Comment (eliperelman) [reply README.md]: Definitely, I'll update.

Comment (eliperelman) [reply README.md]: (that's what I get for copy-pasting :laughing:)

File: package.json [hunk 2]
```diff
@@ -9,6 +9,12 @@
...
   ],
   "dependencies": {
-    "neutrino-preset-react": "^2.1.0"
+    "babel-preset-stage-2": "^6.22.0",
+    "deepmerge": "^1.3.2",
+    "neutrino-preset-airbnb-base": "^4.0.1",
+    "neutrino-preset-react": "^4.0.1"
+  },
+  "peerDependencies": {
+    "neutrino": "^4.0.0"
```
Comment (helfi92) [inline package.json:18]: ^4.0.1

Comment (eliperelman) [reply package.json:18]: This one is a `peerDependency`, the actual version used will be in the dependent project.

File: README.md (hunk 1 above)
Comment (helfi92) [inline README.md]: Ah, you're right

File: package.json (hunk 2 above)
Comment (helfi92) [reply package.json:18]: Ah, you're right

**Comment by helfi92** [comment]
Seems good to me

**Comment by eliperelman** [comment]
Just tested this out in treeherder-manifest, and it worked well. :+1:

**Comment by eliperelman** [comment]
This is now live on npm v2.0.0.

**Pull request merged**